components/sandbox/tiers/staging/appstudio-env/tiertemplate-appstudio-env-admin-37575401-37575401.yaml

```diff
@@ -12,7 +12,8 @@ spec:
   template:
     apiVersion: template.openshift.io/v1
     kind: Template
-    metadata: {}
+    metadata:
+      creationTimestamp: null
```

Comment:
All these changes relate to creationTimestamp should not be here. You need to generate the tiers using  @dperaza4dustbit branch: https://github.com/kubesaw/ksctl/pull/50 of ksctl

Reply:
Done, I re-generated the tiers using that version of ksctl